pilot/pkg/serviceregistry/kube/controller/controller.go

```diff
@@ -646,6 +638,7 @@ func (c *Controller) SyncAll() error {
 
 	err = multierror.Append(err, c.syncPods())
 	err = multierror.Append(err, c.syncEndpoints())
+	err = multierror.Append(err, c.exports.Sync())
```

Comment:
syncEndpoints then exports.Sync will lead to processing all endpoints 2x. This is not needed actually, as far as I can tell, as the syncEndpoints will already call isExported. syncEndpoints only occurs after exports informer has synced so its guaranteed that isExported will function properly